### DISCUSSION TITLE
netlify - Consider for main JB.net deploy previews

### DISCUSSION THREAD

**Discussion by elreydetoda** [JupiterBroadcasting/jupiterbroadcasting.com#182] [Ideas]
So, one thing that I've normally done in the past is hooked up Netlify to the repo. Normally this is for easy of hosting the NetlifyCMS, but a passive added benefit is using them for deploy previews. This means it'll automatically deploy the changes proposed in a PR for the website. It can be used on the free tier and if layer down the road it breaks or cost money. It'll be easy to remove it, and we'll go back to doing what we're currently doing.

So, this is an example past website that has a deploy preview: https://github.com/Climate-Refugee-Stories/crs-website/pull/95

Then this is the config that's used: https://github.com/Climate-Refugee-Stories/crs-website/blob/master/netlify.toml

It'll take someone on the JB side to hook it up (I'd be willing to hop on a call and work through it with them), but it'd be nice to see peoples change *and* interact with them instead of relying on people sending screenshots. 😅

There are more advanced features which you'd have to pay for, i.e. a utility overplayed on the website for screenshot'ing + commenting while you're navigating the deploy preview, but just the actual deploy preview feature is free.

We could eventually hook everything up to do deploy previews based on where the changes were coming from but till then just the JB.net works.

**Comment by reesericci** [comment]
I am personally fully supportive of this as I've also used Netlify on my sites. The main reason I'm supportive though is that it would allow me to deploy the serverless function for the live stream endpoint as detailed in #41

**Comment by reesericci** [comment]
It could also manage the site hosting and redeploy automatically.

**Comment by elreydetoda** [comment]
>It could also manage the site hosting and redeploy automatically.

While yes it could do that, I'd imagine it'd end up costing JB because of how much they'll probably need out of it. So, personally, I'd recommend them just stay with the model they have (GH actions + rsync) and netlify for deploy previews (and serverless functions (if that's possible with our setup)).

I think, at least personally I feel this way, just trying to keep as much that JB can host themselves while not being too much of a maintenance burden or expense is the sweet spot for them.

**Comment by gerbrent** [comment]
great thoughts!
Lets open up this discussion to @StefanS-O and @kbondarev ...

(also converting to a Discussion)

**Reply by elreydetoda** [reply]
It seems like when issues get converted to discussions they die... 🤣😅

**Reply by reesericci** [reply]
yeah...

**Reply by gerbrent** [reply]
hmm!

I wonder why... perhaps they're just outside our everyday usage paradigms.
I will say I do love the ability to make nested comments... that's a beautiful feature I keep wishing for in Issues (not idea why they decided not to also integrate that feature there, seems like a miss.)

Maybe I'll convert them back to issues w a `Discussion` tag - seems there's value in keeping these ideas alive.

**Reply by gerbrent** [reply]
Ugh.
 The Discussion-->Issue process stinks, as we lose most of the additional context/discussion. Maybe that's a feature......

**Reply by elreydetoda** [reply]
Dang, sorry that it isn't very good. It looks like we already had someone comment on one of the converted ones though! So, thank you for migrating them 🚀🥳

Ya, the nested comments is nice! But I guess it's so people don't diverge too much from the topic at hand, or just a limitation 😅

**Comment by gerbrent** [comment]
Conversation on this idea/issue moved to #257. 
Locking this discussion to avoid duplication/divergence...
